src/tests/contract_test/wallet_test.cljs

```diff
@@ -27,32 +28,26 @@
       (p/let [result (contract-utils/call-rpc "accounts_getAccounts")]
         (assert-accounts-get-accounts result)))))
 
-(defn get-default-account
-  [accounts]
-  (first (filter :wallet accounts)))
-
 (defn check-emoji-is-updated
   [test-emoji accounts]
-  (let [default-account (get-default-account accounts)]
+  (let [default-account (contract-utils/get-default-account accounts)]
     (is (= (:emoji default-account) test-emoji))))
 
 (deftest accounts-save-accounts-contract
   (h/test-async :contract/accounts-save-account
     (fn []
       (p/let [test-emoji      (emoji-picker.utils/random-emoji)
               account         (contract-utils/call-rpc "accounts_getAccounts")
-              default-account (get-default-account account)
+              default-account (contract-utils/get-default-account account)
               _ (contract-utils/call-rpc
                  "accounts_saveAccount"
                  (data-store/<-account (merge default-account {:emoji test-emoji})))
               accounts        (contract-utils/call-rpc "accounts_getAccounts")]
         (check-emoji-is-updated test-emoji accounts)))))
 
-(def number-of-networks 3)
```

Comment:
imo this is better defined in the test file. Do really need this defined as a global constant? 🤔